src/book.js

```diff
@@ -0,0 +1,17 @@
+export default function book(){
+    return `
+    <div>
+        <h1>Book</h1>
+        <h2>Author</h2>
+        <h3>Price</h3>   
+        <h3>Selling Points</h3>
+            <ol>
```

Comment:
The markup structure looks much better, but there are still nesting/`TAB`-ing issues issues with the `ol` and `img` elements.